SWIFT/day16.swift

```diff
@@ -0,0 +1,51 @@
+import Foundation
+
+// Class Difference
+class Difference {
+
+  // MARK: Private Instance variables
+  private var elements          = [Int]()
+  private var minimumPairs      = [Int]()
+  private var minimumDifference = 0
```

Comment:
Is there a test case that will cause the wrong output if initial diff is set to 0?